pkg/errinsrc/testdata/Test_renderSrc_Simple__single_character_error_ascii.golden

```diff
@@ -3,16 +3,16 @@
 
 There has been a simple error in your code
 
-    ,-[ /test.cue:13:13 ]
-    |
- 11 | 
- 12 | // If foo then bar is 12
- 13 | if blah.foo {
-    :             ^
-    :             `- This brace is dodgy
- 14 | 	blah: bar: 12
- 15 | }
-----'
+   ,-[ /test.cue:6:11 ]
+   |
+ 4 | blah: {
+ 5 |     foo: bool
+ 6 |     bar: int | *3
+   :              ^
+   :              `- This looks dodgy
```

Comment:
I changed this test, because on the `{` it now picked up the block and put the end at line 15